samples/samples-powershell/AddProductParams/run.ps1

```diff
@@ -1,16 +1,18 @@
 using namespace System.Net
 
 # Trigger binding data passed in via param block
-param($Request)
+param($Request, $TriggerMetadata)
 
 # Write to the Azure Functions log stream.
 Write-Host "PowerShell function with SQL Output Binding processed a request."
 
 # Update req_query with the query of the request
+# Currently use a workaround through the TriggerMetadata for PowerShell Azure Functions
+# Issue link: https://github.com/Azure/azure-functions-powershell-worker/issues/895
```

Comment:
Please also make a tracking issue in our repo to remove this workaround once they've fixed it in the worker.

Reply:
Tracked here: https://github.com/Azure/azure-functions-sql-extension/issues/535